modules/core/src/main/resources/artifact-migrations.conf

```
@@ -22,5 +22,11 @@ changes = [
     groupIdAfter = org.typelevel
     artifactIdAfter = kind-projector
     initialVersion = 0.10.0
+  },
+  {
+    groupIdBefore = org.scalatestplus
```

```suggestion
    groupIdAfter = org.scalatestplus
```
It's the other way around: https://github.com/scala-steward-org/scala-steward/blob/master/docs/artifact-migrations.md
The `*After` fields are mandantory.